test/built-ins/Temporal/PlainDate/prototype/with/order-of-operations.js

```diff
@@ -15,6 +15,15 @@ const expected = [
   // CalendarFields
   "get this.calendar.fields",
   "call this.calendar.fields",
+  // PrepareTemporalFields on receiver
+  "get this.calendar.day",
+  "call this.calendar.day",
+  "get this.calendar.month",
+  "call this.calendar.month",
+  "get this.calendar.monthCode",
+  "call this.calendar.monthCode",
+  "get this.calendar.year",
+  "call this.calendar.year",
   // PrepareTemporalFields
```

Comment:
```suggestion
  // PrepareTemporalFields on argument
```
(here and elsewhere)